server/controllers/chartsController/helpers.test.js

```diff
@@ -1009,4 +1009,91 @@ describe('chartsController - helpers', () => {
       ])
     })
   })
+
+  describe(helpers.graphTableColumns, () => {
+    it('returns an array of table headers from a list of labels', () => {
+      expect(helpers.graphTableColumns(helpersFactories.labels)).toEqual([
+        'Site',
+        'Pending evaluation',
+        'Excellent',
+        'Good',
+        'Average',
+        'Poor',
+        'Total',
+      ])
+    })
+
+    it('removes the N/A category from the labels list', () => {
+      const isNAValueInArray = helpers
+        .graphTableColumns(helpersFactories.labels)
+        .some((label) => label === N_A)
+      expect(isNAValueInArray).toEqual(false)
+    })
+
+    it('prepends the header "Site" to array', () => {
+      expect(helpers.graphTableColumns(helpersFactories.labels)[0]).toEqual(
+        'Site'
+      )
+    })
```

Comment:
isn't this covered in the expectation on line 1015?

Reply:
https://github.com/AMP-SCZ/dpdash/commit/53c75c19215f81ea1f646fb5e98102019a21521f